world/server.go

```diff
@@ -49,6 +49,10 @@ func (server *Server) ServerDisconnected(conn mnet.Server) {
 	server.login.Send(server.info.GenerateInfoPacket())
 }
 
+func (server *Server) FillRates(exp, drop, mesos int) {
```

Comment:
This function is not needed and what it does can just be inline.

Reply:
i put this in a method because doing it inline i will need to export the 'info' field when all other fields are not

Reply:
I'm ok with making the info field exported since all of it's member variables are, but I will leave it up to you.